sd_source/Subs-SimpleDesk.php

```diff
@@ -1129,7 +1129,7 @@ function shd_recalc_ids($ticket)
 			INNER JOIN {db_prefix}helpdesk_tickets AS hdt ON (hdt.id_ticket = hdtr.id_ticket)
 		WHERE hdtr.id_msg > hdt.id_first_msg
 			AND hdt.id_ticket = {int:ticket}
-		GROUP BY hdtr.message_status',
+		GROUP BY hdtr.message_status, hdt.id_first_msg',
 		array(
```

Comment:
What will this fix?

Reply:
the query got an error,
because the col hdt.id_first_msg is not part of aggragation function and is not in group by term.

so i fixed this by adding this to the group by part.